chapters/functions.tex

```diff
@@ -116,6 +116,29 @@ \subsection{Inheritance of Functions}\label{inheritance-of-functions}
 For example, it is possible to modify and extend a \lstinline!function! class to add default values for input variables.
 \end{nonnormative}
 
+A special case is defining a \lstinline!function! as a short-class definition with modifiers for inputs inside a model.
+These default values, unless overridden in the function call, will then be considered for variability similarly as if they were given in the function call.
```

Comment:
It would be good to have a cross reference to the new _Function Variability_ section here.

Reply:
Done

Reply:
OK.  Resolving.